spec/helpers/pagination_helper_spec.rb

```diff
@@ -81,4 +83,24 @@
       expect(@test_class.start_index(5)).to eq(41)
     end
   end
+
+  context '#active_tile' do
+    it 'checks if active page is tile 3' do
+      @test_class.instance_variable_set(:@start_index, 21)
+      @test_class.instance_variable_set(:@results_total, 536)
+      expect(@test_class.active_tile).to eq(3)
+    end
+
+    it 'checks if active page is tile 6' do
+      @test_class.instance_variable_set(:@start_index, 61)
+      @test_class.instance_variable_set(:@results_total, 536)
+      expect(@test_class.active_tile).to eq(6)
+    end
+
+    it 'checks if active page is tile 8' do
```

Comment:
Does `it` want to contextualise this - `tile 8 between 511 and 536`, for instance?

Reply:
`it` contextualizes the active tile, which you can view on line 87, which is `context '#active_tile' do`.